docs/snippets/all/archetypes/points3d_send_columns.rs

```diff
@@ -0,0 +1,37 @@
+//! Use the `send_columns` API to send several point clouds over time in a single call.
+
+use rerun::TimeColumn;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_points3d_send_columns").spawn()?;
+
+    // Prepare a point cloud that evolves over 5 timesteps, changing the number of points in the process.
+    let times = TimeColumn::new_seconds("time", 10..15);
+
+    #[rustfmt::skip]
+    let positions = || [
+        vec![[1.0, 0.0, 1.0], [0.5, 0.5, 2.0]],
+        vec![[1.5, -0.5, 1.5], [1.0, 1.0, 2.5], [-0.5, 1.5, 1.0], [-1.5, 0.0, 2.0]],
+        vec![[2.0, 0.0, 2.0], [1.5, -1.5, 3.0], [0.0, -2.0, 2.5], [1.0, -1.0, 3.5]],
+        vec![[-2.0, 0.0, 2.0], [-1.5, 1.5, 3.0], [-1.0, 1.0, 3.5]],
+        vec![[1.0, -1.0, 1.0], [2.0, -2.0, 2.0], [3.0, -1.0, 3.0], [2.0, 0.0, 4.0]],
+    ]
```

Comment:
I don't like the way of writing the example this way much. `send_columns` is all about having flat data to begin with and if your data were to look like this then it wouldn't be a good fit.
This is why back then I went with a flat position list in C++ and then provided the partitions separately
https://github.com/rerun-io/rerun/pull/8764/files#diff-4b7c039da83394b4869f80ddbaacbcf555c70653dc990c6a1609ccb8eb3a326cR36

Reply:
Ha yeah I didn't notice it was flat, the way this was formatted made me think it wasn't. I'll update.